internal/jsre/deps/web3.js

```diff
@@ -3715,6 +3715,21 @@ var inputBlockNumberFormatter = function (blockNumber) {
     return utils.toHex(blockNumber);
 };
 
+var inputTraceFormatter = function (config) {
+    if (config !== undefined && Object.keys(config).length > 0) {
+        var validParams = collectTraceConfigFields().split(",");
+        var copied = {};
+        for (var field in config) copied[field.toLowerCase()] = config[field];
```

Comment:
Please use brackets to make the scope clear: 
```suggestion
        for (var field in config){ copied[field.toLowerCase()] = config[field]; }
```
Next line too

Reply:
Added scopes.